openlmis-stock/src/main/java/org/smartregister/stock/openlmis/util/OpenLMISConstants.java

```diff
@@ -36,6 +36,20 @@
 
     String PREV_SYNC_SERVER_VERSION_ORDERABLE = "prev_sync_server_version_orderable";
 
+    String IS_REMOTE_LOGIN = "is_remote_login";
+
+    String ENGLISH_LANGUAGE = "English";
+
+    String URDU_LANGUAGE = "Urdu";
+
+    String ENGLISH_LOCALE = "en";
+
```

Comment:
Whats the purpose of 
   `String URDU_LANGUAGE = "Urdu";`
     `String ENGLISH_LOCALE = "en";`
     `String URDU_LOCALE = "ur";`